src/ProjectTemplates/test/ProjectTemplates.Tests.csproj

```diff
@@ -8,7 +8,7 @@
     <TestGroupName>ProjectTemplates.E2ETests</TestGroupName>
     <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
 
-    <RunTemplateTests>true</RunTemplateTests>
+    <RunTemplateTests Condition="'$(RunTemplateTests)' == ''" >true</RunTemplateTests>
```

Comment:
`?` These tests already ran in all the platforms. Are you trying to run the Selenium bits? (which only run on windows)

Reply:
If that's the case `SeleniumE2ETestsSupported` that's what you are looking to set

Reply:
No, I'm just trying to clarify that `RunTemplateTests` can be (and is) set elsewhere, and that should be the value if it is.